Makefile

```diff
@@ -1,5 +1,5 @@
 # Current Operator version
-VERSION ?= 0.20.0
```

Comment:
shouldn't update the version

Reply:
why? this needs to be released?

Reply:
i have updated the PR but i would like to know what the process is here or where it's documented as i dont understand why we would not want to release this

Reply:
here is the release process of FunctionMesh: https://www.notion.so/streamnativeio/Function-Mesh-Release-Process-68e7da640d81411092bd54f974c505cb

only when need to do a release for FunctionMesh, this value should be updated